tests/10apidoc/00prepare.pl

```diff
@@ -26,3 +28,19 @@ sub do_request_json_for
       %args,
    );
 }
+
+
+sub new_User
```

Comment:
On second inspection, that would complicate the optional nature of most of the parameters and the defaulting arrayref for `saved_events`; so may be best to leave it as it is.